2024-09-18 10:41:47,892 - app - INFO - Starting the application
2024-09-18 10:41:47,892 - app - INFO - Configuration loaded from configuration/config.json
2024-09-18 10:41:47,920 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-18 10:41:47,920 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 10:41:47,920 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-18 10:41:47,920 - app - INFO - Attempting to connect to hostname: itsaplasticbottle.com
2024-09-18 10:41:47,925 - app - INFO - Resolved itsaplasticbottle.com to 35.222.24.210
2024-09-18 10:41:48,101 - app - INFO - TCP connection established
2024-09-18 10:41:48,171 - app - INFO - SSL/TLS handshake completed
2024-09-18 10:41:48,171 - app - INFO - Certificate successfully retrieved
2024-09-18 10:41:48,172 - app - INFO - Initial certificate pinned with hash: 4bfb33fb101ab62c9e41193926d81273707daf139c1e4a607ad14107d0ff5ed5
2024-09-18 10:41:48,176 - app - INFO - Initialized AuthManager for environment: prod
2024-09-18 10:41:48,176 - app - INFO - Using urls wss://itsaplasticbottle.com/sockets/device, https://itsaplasticbottle.com/api/auth/device_auth
2024-09-18 10:41:48,176 - app - INFO - Setting Mode to idle
2024-09-18 10:41:48,176 - app - INFO - No change in Mode
2024-09-18 10:41:48,682 - app - ERROR - Error updating certificate hash: startswith first arg must be bytes or a tuple of bytes, not str
2024-09-18 10:41:48,683 - app - INFO - Successfully obtained new JWT token
2024-09-18 10:41:49,041 - app - INFO - Connected to WebSocket server
2024-09-18 10:41:49,042 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 10:42:00,516 - app - INFO - Setting Mode to stream_only
2024-09-18 10:42:00,517 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 10:42:00,517 - app - INFO - Tracker Mode is stream_only
2024-09-18 10:42:00,517 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-18 10:42:00,517 - app - INFO - Initialized StreamManager with output URL: rtmp://itsaplasticbottle.com/live/11?key=Vf4C71LF7gsaRPAW1v-UcJW4fnOn7e50THXJgZMB-f8
2024-09-18 10:42:00,517 - app - INFO - Starting stream
2024-09-18 10:42:00,518 - app - INFO - Stream started successfully
2024-09-18 10:42:00,518 - app - INFO - succesfully started stream manager
2024-09-18 10:42:00,518 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-18 10:42:00,642 - app - INFO - Initialized TrackerManager with mode: stream_only
2024-09-18 10:42:00,643 - app - INFO - Mode set to stream_only
2024-09-18 10:42:00,643 - app - INFO - Starting tracking in mode: stream_only
2024-09-18 10:42:57,550 - app - INFO - Setting Mode to idle
2024-09-18 10:42:57,550 - app - INFO - Stream stopped successfully
2024-09-18 10:42:57,550 - app - INFO - Stopped TrackerManager
2024-09-18 10:42:57,550 - app - INFO - Tracking task cancelled
2024-09-18 10:42:57,550 - app - WARNING - No active stream to stop
2024-09-18 10:42:57,550 - app - INFO - Stopped TrackerManager
2024-09-18 10:42:57,550 - app - INFO - Cleaned up TrackerManager resources
2024-09-18 10:42:57,551 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 10:42:57,551 - app - INFO - Mode set to idle
2024-09-18 10:59:36,233 - app - INFO - Cleaning up resources...
2024-09-18 10:59:36,300 - app - INFO - Disconnected from WebSocket server
2024-09-18 10:59:36,300 - app - INFO - AuthManager cleaned up
2024-09-18 12:24:20,345 - app - INFO - Starting the application
2024-09-18 12:24:20,345 - app - INFO - Configuration loaded from configuration/config.json
2024-09-18 12:24:20,346 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-18 12:24:20,346 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 12:24:20,346 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-18 12:24:20,346 - app - INFO - Attempting to connect to hostname: itsaplasticbottle.com
2024-09-18 12:24:20,474 - app - INFO - Resolved itsaplasticbottle.com to 35.222.24.210
2024-09-18 12:24:20,626 - app - INFO - TCP connection established
2024-09-18 12:24:20,703 - app - INFO - SSL/TLS handshake completed
2024-09-18 12:24:20,703 - app - INFO - Certificate successfully retrieved
2024-09-18 12:24:20,704 - app - INFO - Initial certificate pinned with hash: 4bfb33fb101ab62c9e41193926d81273707daf139c1e4a607ad14107d0ff5ed5
2024-09-18 12:24:20,709 - app - INFO - Initialized AuthManager for environment: prod
2024-09-18 12:24:20,709 - app - INFO - Using urls wss://itsaplasticbottle.com/sockets/device, https://itsaplasticbottle.com/api/auth/device_auth
2024-09-18 12:24:20,709 - app - INFO - Setting Mode to idle
2024-09-18 12:24:20,709 - app - INFO - No change in Mode
2024-09-18 12:24:21,290 - app - ERROR - Error updating certificate hash: startswith first arg must be bytes or a tuple of bytes, not str
2024-09-18 12:24:21,290 - app - ERROR - Failed to authenticate device: 401
2024-09-18 12:24:21,290 - app - ERROR - Error during authentication: Authentication failed with status 401
2024-09-18 12:24:21,291 - app - ERROR - 
2024-09-18 12:24:25,144 - app - INFO - Cleaning up resources...
2024-09-18 12:24:25,144 - app - INFO - Disconnected from WebSocket server
2024-09-18 12:24:25,144 - app - INFO - AuthManager cleaned up
2024-09-18 12:24:41,764 - app - INFO - Starting the application
2024-09-18 12:24:41,764 - app - INFO - Configuration loaded from configuration/config.json
2024-09-18 12:24:41,765 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-18 12:24:41,765 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 12:24:41,765 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-18 12:24:41,765 - app - INFO - Attempting to connect to hostname: localhost
2024-09-18 12:24:41,769 - app - INFO - Resolved localhost to 127.0.0.1
2024-09-18 12:24:41,770 - app - ERROR - Failed to get server certificate: [Errno 111] Connection refused
2024-09-18 12:24:41,770 - app - ERROR - Failed to perform initial certificate pinning: [Errno 111] Connection refused
2024-09-18 12:25:21,756 - app - INFO - Starting the application
2024-09-18 12:25:21,756 - app - INFO - Configuration loaded from configuration/config.json
2024-09-18 12:25:21,757 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-18 12:25:21,757 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 12:25:21,757 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-18 12:25:21,757 - app - INFO - Attempting to connect to hostname: localhost
2024-09-18 12:25:21,762 - app - INFO - Resolved localhost to 127.0.0.1
2024-09-18 12:25:21,762 - app - ERROR - Failed to get server certificate: [Errno 111] Connection refused
2024-09-18 12:25:21,762 - app - ERROR - Failed to perform initial certificate pinning: [Errno 111] Connection refused
2024-09-18 12:25:48,270 - app - INFO - Starting the application
2024-09-18 12:25:48,271 - app - INFO - Configuration loaded from configuration/config.json
2024-09-18 12:25:48,271 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-18 12:25:48,271 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 12:25:48,272 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-18 12:25:48,272 - app - INFO - Attempting to connect to hostname: localhost
2024-09-18 12:25:48,276 - app - INFO - Resolved localhost to 127.0.0.1
2024-09-18 12:25:48,276 - app - ERROR - Failed to get server certificate: [Errno 111] Connection refused
2024-09-18 12:25:48,276 - app - ERROR - Failed to perform initial certificate pinning: [Errno 111] Connection refused
2024-09-18 12:26:28,983 - app - INFO - Starting the application
2024-09-18 12:26:28,983 - app - INFO - Configuration loaded from configuration/config.json
2024-09-18 12:26:28,984 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-18 12:26:28,984 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 12:26:28,984 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-18 12:26:28,984 - app - INFO - Development environment detected. Skipping SSL configuration.
2024-09-18 12:26:28,984 - app - INFO - Initialized AuthManager for environment: dev
2024-09-18 12:26:28,984 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-18 12:26:28,984 - app - INFO - Setting Mode to idle
2024-09-18 12:26:28,984 - app - INFO - No change in Mode
2024-09-18 12:26:29,040 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:26:29,041 - app - INFO - Successfully obtained new JWT token
2024-09-18 12:26:29,075 - app - INFO - Connected to WebSocket server
2024-09-18 12:26:29,075 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:28:19,458 - app - INFO - Cleaning up resources...
2024-09-18 12:28:19,458 - app - INFO - Disconnected from WebSocket server
2024-09-18 12:28:19,459 - app - INFO - AuthManager cleaned up
2024-09-18 12:28:21,950 - app - INFO - Starting the application
2024-09-18 12:28:21,950 - app - INFO - Configuration loaded from configuration/config.json
2024-09-18 12:28:21,951 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-18 12:28:21,951 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 12:28:21,951 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-18 12:28:21,951 - app - INFO - Development environment detected. Skipping SSL configuration.
2024-09-18 12:28:21,951 - app - INFO - Initialized AuthManager for environment: dev
2024-09-18 12:28:21,951 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-18 12:28:21,951 - app - INFO - Setting Mode to idle
2024-09-18 12:28:21,951 - app - INFO - No change in Mode
2024-09-18 12:28:22,018 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:28:22,018 - app - INFO - Successfully obtained new JWT token
2024-09-18 12:28:22,052 - app - INFO - Connected to WebSocket server
2024-09-18 12:28:22,052 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:29:23,076 - app - INFO - Cleaning up resources...
2024-09-18 12:29:23,076 - app - INFO - Disconnected from WebSocket server
2024-09-18 12:29:23,077 - app - INFO - AuthManager cleaned up
2024-09-18 12:29:25,575 - app - INFO - Starting the application
2024-09-18 12:29:25,576 - app - INFO - Configuration loaded from configuration/config.json
2024-09-18 12:29:25,576 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-18 12:29:25,577 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 12:29:25,577 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-18 12:29:25,577 - app - INFO - Development environment detected. Skipping SSL configuration.
2024-09-18 12:29:25,577 - app - INFO - Initialized AuthManager for environment: dev
2024-09-18 12:29:25,577 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-18 12:29:25,577 - app - INFO - Setting Mode to idle
2024-09-18 12:29:25,577 - app - INFO - No change in Mode
2024-09-18 12:29:25,651 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:29:25,652 - app - INFO - Successfully obtained new JWT token
2024-09-18 12:29:25,686 - app - INFO - Connected to WebSocket server
2024-09-18 12:29:25,686 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:30:17,912 - app - INFO - Cleaning up resources...
2024-09-18 12:30:17,913 - app - INFO - Disconnected from WebSocket server
2024-09-18 12:30:17,913 - app - INFO - AuthManager cleaned up
2024-09-18 12:30:20,322 - app - INFO - Starting the application
2024-09-18 12:30:20,322 - app - INFO - Configuration loaded from configuration/config.json
2024-09-18 12:30:20,323 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-18 12:30:20,323 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 12:30:20,323 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-18 12:30:20,323 - app - INFO - Development environment detected. Skipping SSL configuration.
2024-09-18 12:30:20,323 - app - INFO - Initialized AuthManager for environment: dev
2024-09-18 12:30:20,323 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-18 12:30:20,323 - app - INFO - Setting Mode to idle
2024-09-18 12:30:20,323 - app - INFO - No change in Mode
2024-09-18 12:30:20,384 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:30:20,384 - app - INFO - Successfully obtained new JWT token
2024-09-18 12:30:20,418 - app - INFO - Connected to WebSocket server
2024-09-18 12:30:20,419 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:30:25,325 - app - INFO - received stream command True
2024-09-18 12:31:09,642 - app - INFO - Cleaning up resources...
2024-09-18 12:31:09,643 - app - INFO - Disconnected from WebSocket server
2024-09-18 12:31:09,644 - app - INFO - AuthManager cleaned up
2024-09-18 12:31:11,873 - app - INFO - Starting the application
2024-09-18 12:31:11,873 - app - INFO - Configuration loaded from configuration/config.json
2024-09-18 12:31:11,874 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-18 12:31:11,874 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 12:31:11,874 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-18 12:31:11,874 - app - INFO - Development environment detected. Skipping SSL configuration.
2024-09-18 12:31:11,874 - app - INFO - Initialized AuthManager for environment: dev
2024-09-18 12:31:11,874 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-18 12:31:11,874 - app - INFO - Setting Mode to idle
2024-09-18 12:31:11,875 - app - INFO - No change in Mode
2024-09-18 12:31:11,934 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:31:11,934 - app - INFO - Successfully obtained new JWT token
2024-09-18 12:31:11,969 - app - INFO - Connected to WebSocket server
2024-09-18 12:31:11,969 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:31:18,743 - app - INFO - received stream command True
2024-09-18 12:31:57,827 - app - INFO - Cleaning up resources...
2024-09-18 12:31:57,828 - app - INFO - Disconnected from WebSocket server
2024-09-18 12:31:57,829 - app - INFO - AuthManager cleaned up
2024-09-18 12:32:00,308 - app - INFO - Starting the application
2024-09-18 12:32:00,308 - app - INFO - Configuration loaded from configuration/config.json
2024-09-18 12:32:00,309 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-18 12:32:00,309 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 12:32:00,309 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-18 12:32:00,309 - app - INFO - Development environment detected. Skipping SSL configuration.
2024-09-18 12:32:00,309 - app - INFO - Initialized AuthManager for environment: dev
2024-09-18 12:32:00,309 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-18 12:32:00,309 - app - INFO - Setting Mode to idle
2024-09-18 12:32:00,309 - app - INFO - No change in Mode
2024-09-18 12:32:00,369 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:32:00,369 - app - INFO - Successfully obtained new JWT token
2024-09-18 12:32:00,404 - app - INFO - Connected to WebSocket server
2024-09-18 12:32:00,404 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:32:05,312 - app - INFO - received stream command True
2024-09-18 12:32:23,466 - app - INFO - Cleaning up resources...
2024-09-18 12:32:23,467 - app - INFO - Disconnected from WebSocket server
2024-09-18 12:32:23,467 - app - INFO - AuthManager cleaned up
2024-09-18 12:32:25,941 - app - INFO - Starting the application
2024-09-18 12:32:25,942 - app - INFO - Configuration loaded from configuration/config.json
2024-09-18 12:32:25,942 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-18 12:32:25,942 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 12:32:25,942 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-18 12:32:25,943 - app - INFO - Development environment detected. Skipping SSL configuration.
2024-09-18 12:32:25,943 - app - INFO - Initialized AuthManager for environment: dev
2024-09-18 12:32:25,943 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-18 12:32:25,943 - app - INFO - Setting Mode to idle
2024-09-18 12:32:25,943 - app - INFO - No change in Mode
2024-09-18 12:32:26,007 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:32:26,007 - app - INFO - Successfully obtained new JWT token
2024-09-18 12:32:26,041 - app - INFO - Connected to WebSocket server
2024-09-18 12:32:26,041 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:32:30,945 - app - INFO - received stream command True
2024-09-18 12:33:45,375 - app - INFO - Cleaning up resources...
2024-09-18 12:33:45,376 - app - INFO - Disconnected from WebSocket server
2024-09-18 12:33:45,376 - app - INFO - AuthManager cleaned up
2024-09-18 12:33:47,917 - app - INFO - Starting the application
2024-09-18 12:33:47,917 - app - INFO - Configuration loaded from configuration/config.json
2024-09-18 12:33:47,918 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-18 12:33:47,918 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 12:33:47,918 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-18 12:33:47,918 - app - INFO - Development environment detected. Skipping SSL configuration.
2024-09-18 12:33:47,918 - app - INFO - Initialized AuthManager for environment: dev
2024-09-18 12:33:47,918 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-18 12:33:47,918 - app - INFO - Setting Mode to idle
2024-09-18 12:33:47,918 - app - INFO - No change in Mode
2024-09-18 12:33:47,992 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:33:47,992 - app - INFO - Successfully obtained new JWT token
2024-09-18 12:33:48,026 - app - INFO - Connected to WebSocket server
2024-09-18 12:33:48,027 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:33:52,920 - app - INFO - received stream command True
2024-09-18 12:33:52,920 - app - INFO - stream is true
2024-09-18 12:34:15,661 - app - INFO - Cleaning up resources...
2024-09-18 12:34:15,662 - app - INFO - Disconnected from WebSocket server
2024-09-18 12:34:15,662 - app - INFO - AuthManager cleaned up
2024-09-18 12:39:02,881 - app - INFO - Starting the application
2024-09-18 12:39:02,881 - app - INFO - Configuration loaded from configuration/config.json
2024-09-18 12:39:02,882 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-18 12:39:02,882 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 12:39:02,882 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-18 12:39:02,882 - app - INFO - Development environment detected. Skipping SSL configuration.
2024-09-18 12:39:02,882 - app - INFO - Initialized AuthManager for environment: dev
2024-09-18 12:39:02,882 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-18 12:39:02,882 - app - INFO - Setting Mode to idle
2024-09-18 12:39:02,882 - app - INFO - No change in Mode
2024-09-18 12:39:02,943 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:39:02,943 - app - INFO - Successfully obtained new JWT token
2024-09-18 12:39:02,976 - app - INFO - Connected to WebSocket server
2024-09-18 12:39:02,977 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:39:08,160 - app - INFO - Setting Mode to collect_data
2024-09-18 12:39:08,161 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:39:08,220 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:39:08,220 - app - INFO - Tracker Mode is collect_data
2024-09-18 12:39:08,220 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-18 12:39:08,345 - app - INFO - Initialized TrackerManager with mode: collect_data
2024-09-18 12:39:08,345 - app - INFO - Mode set to collect_data
2024-09-18 12:39:08,346 - app - INFO - Starting tracking in mode: collect_data
2024-09-18 12:39:18,368 - app - INFO - Sending data from tracking loop
2024-09-18 12:39:18,405 - app - ERROR - Error uploading: Constructor parameter should be str
2024-09-18 12:39:19,443 - app - INFO - Cleaning up resources...
2024-09-18 12:39:19,444 - app - INFO - Stopped TrackerManager
2024-09-18 12:39:19,444 - app - INFO - Cleaned up TrackerManager resources
2024-09-18 12:39:19,444 - app - INFO - Disconnected from WebSocket server
2024-09-18 12:39:19,445 - app - INFO - AuthManager cleaned up
2024-09-18 12:39:19,445 - app - INFO - Tracking task cancelled
2024-09-18 12:39:19,445 - app - INFO - Stopped TrackerManager
2024-09-18 12:39:19,445 - app - INFO - Cleaned up TrackerManager resources
2024-09-18 12:46:50,098 - app - INFO - Starting the application
2024-09-18 12:46:50,099 - app - INFO - Configuration loaded from configuration/config.json
2024-09-18 12:46:50,099 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-18 12:46:50,099 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 12:46:50,099 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 12:46:50,100 - app - INFO - Development environment detected. Skipping SSL configuration.
2024-09-18 12:46:50,100 - app - INFO - Initialized AuthManager for environment: dev
2024-09-18 12:46:50,100 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-18 12:46:50,100 - app - INFO - Setting Mode to collect_data
2024-09-18 12:46:50,100 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:46:50,148 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:46:50,148 - app - INFO - Tracker Mode is collect_data
2024-09-18 12:46:50,148 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-18 12:46:50,273 - app - INFO - Initialized TrackerManager with mode: collect_data
2024-09-18 12:46:50,273 - app - INFO - Mode set to collect_data
2024-09-18 12:46:50,274 - app - INFO - Starting tracking in mode: collect_data
2024-09-18 12:46:50,343 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:46:50,343 - app - INFO - Successfully obtained new JWT token
2024-09-18 12:46:50,379 - app - INFO - Connected to WebSocket server
2024-09-18 12:46:50,380 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:46:58,410 - app - INFO - Setting Mode to stream_only
2024-09-18 12:46:58,410 - app - INFO - Stopped TrackerManager
2024-09-18 12:46:58,411 - app - INFO - Tracking task cancelled
2024-09-18 12:46:58,411 - app - INFO - Stopped TrackerManager
2024-09-18 12:46:58,411 - app - INFO - Cleaned up TrackerManager resources
2024-09-18 12:46:58,477 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:46:58,477 - app - INFO - Tracker Mode is stream_only
2024-09-18 12:46:58,477 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-18 12:46:58,477 - app - INFO - Initialized StreamManager with output URL: rtmp://itsaplasticbottle.com/live/22?key=cmOm22JhNULO8NGMfkNmsjZOFu6tTqL5pi2RG4OjH0M
2024-09-18 12:46:58,477 - app - INFO - Starting stream
2024-09-18 12:46:58,478 - app - INFO - Stream started successfully
2024-09-18 12:46:58,478 - app - INFO - succesfully started stream manager
2024-09-18 12:46:58,478 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-18 12:46:58,503 - app - INFO - Initialized TrackerManager with mode: stream_only
2024-09-18 12:46:58,503 - app - INFO - Mode set to stream_only
2024-09-18 12:46:58,504 - app - INFO - Starting tracking in mode: stream_only
2024-09-18 12:46:59,806 - app - ERROR - Broken pipe detected. Attempting to restart stream.
2024-09-18 12:46:59,807 - app - INFO - Attempting to reconnect...
2024-09-18 12:46:59,807 - app - INFO - Stream stopped successfully
2024-09-18 12:47:01,808 - app - INFO - Starting stream
2024-09-18 12:47:01,809 - app - INFO - Stream started successfully
2024-09-18 12:47:02,919 - app - ERROR - Broken pipe detected. Attempting to restart stream.
2024-09-18 12:47:02,920 - app - INFO - Attempting to reconnect...
2024-09-18 12:47:02,920 - app - INFO - Stream stopped successfully
2024-09-18 12:47:03,005 - app - INFO - received stream command True
2024-09-18 12:47:03,005 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-18 12:47:03,005 - app - INFO - Initialized StreamManager with output URL: rtmp://itsaplasticbottle.com/live/22?key=cmOm22JhNULO8NGMfkNmsjZOFu6tTqL5pi2RG4OjH0M
2024-09-18 12:47:03,005 - app - INFO - Starting stream
2024-09-18 12:47:03,006 - app - INFO - Stream started successfully
2024-09-18 12:47:03,006 - app - INFO - succesfully started stream manager
2024-09-18 12:47:04,921 - app - INFO - Starting stream
2024-09-18 12:47:04,922 - app - INFO - Stream started successfully
2024-09-18 12:47:05,890 - app - ERROR - Broken pipe detected. Attempting to restart stream.
2024-09-18 12:47:05,891 - app - INFO - Attempting to reconnect...
2024-09-18 12:47:05,891 - app - INFO - Stream stopped successfully
2024-09-18 12:47:07,891 - app - INFO - Starting stream
2024-09-18 12:47:07,893 - app - INFO - Stream started successfully
2024-09-18 12:47:08,841 - app - ERROR - Broken pipe detected. Attempting to restart stream.
2024-09-18 12:47:08,841 - app - INFO - Attempting to reconnect...
2024-09-18 12:47:08,841 - app - INFO - Stream stopped successfully
2024-09-18 12:47:10,842 - app - INFO - Starting stream
2024-09-18 12:47:10,843 - app - INFO - Stream started successfully
2024-09-18 12:47:11,788 - app - ERROR - Broken pipe detected. Attempting to restart stream.
2024-09-18 12:47:11,788 - app - INFO - Attempting to reconnect...
2024-09-18 12:47:11,788 - app - INFO - Stream stopped successfully
2024-09-18 12:47:13,790 - app - INFO - Starting stream
2024-09-18 12:47:13,791 - app - INFO - Stream started successfully
2024-09-18 12:47:14,745 - app - ERROR - Broken pipe detected. Attempting to restart stream.
2024-09-18 12:47:14,745 - app - INFO - Attempting to reconnect...
2024-09-18 12:47:14,745 - app - INFO - Stream stopped successfully
2024-09-18 12:47:16,747 - app - INFO - Starting stream
2024-09-18 12:47:16,748 - app - INFO - Stream started successfully
2024-09-18 12:47:17,907 - app - ERROR - Broken pipe detected. Attempting to restart stream.
2024-09-18 12:47:17,908 - app - INFO - Attempting to reconnect...
2024-09-18 12:47:17,908 - app - INFO - Stream stopped successfully
2024-09-18 12:47:19,909 - app - INFO - Starting stream
2024-09-18 12:47:19,910 - app - INFO - Stream started successfully
2024-09-18 12:47:21,135 - app - ERROR - Broken pipe detected. Attempting to restart stream.
2024-09-18 12:47:21,135 - app - INFO - Attempting to reconnect...
2024-09-18 12:47:21,136 - app - INFO - Stream stopped successfully
2024-09-18 12:47:23,137 - app - INFO - Starting stream
2024-09-18 12:47:23,138 - app - INFO - Stream started successfully
2024-09-18 12:47:24,341 - app - ERROR - Broken pipe detected. Attempting to restart stream.
2024-09-18 12:47:24,342 - app - INFO - Attempting to reconnect...
2024-09-18 12:47:24,342 - app - INFO - Stream stopped successfully
2024-09-18 12:47:24,559 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:47:24,559 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 12:47:24,559 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 12:47:24,559 - app - INFO - Setting Mode to idle
2024-09-18 12:47:24,559 - app - WARNING - No active stream to stop
2024-09-18 12:47:24,664 - app - INFO - Stopped TrackerManager
2024-09-18 12:47:24,664 - app - INFO - Tracking task cancelled
2024-09-18 12:47:24,665 - app - WARNING - No active stream to stop
2024-09-18 12:47:24,665 - app - INFO - Stopped TrackerManager
2024-09-18 12:47:24,665 - app - INFO - Cleaned up TrackerManager resources
2024-09-18 12:47:24,665 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:47:24,665 - app - INFO - Mode set to idle
2024-09-18 12:47:25,166 - app - INFO - Setting Mode to stream_only
2024-09-18 12:47:25,166 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:47:25,166 - app - INFO - Tracker Mode is stream_only
2024-09-18 12:47:25,166 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-18 12:47:25,166 - app - INFO - Initialized StreamManager with output URL: rtmp://localhost/live/22?key=cmOm22JhNULO8NGMfkNmsjZOFu6tTqL5pi2RG4OjH0M
2024-09-18 12:47:25,167 - app - INFO - Starting stream
2024-09-18 12:47:25,167 - app - INFO - Stream started successfully
2024-09-18 12:47:25,167 - app - INFO - succesfully started stream manager
2024-09-18 12:47:25,167 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-18 12:47:25,192 - app - INFO - Initialized TrackerManager with mode: stream_only
2024-09-18 12:47:25,193 - app - INFO - Mode set to stream_only
2024-09-18 12:47:25,193 - app - INFO - Starting tracking in mode: stream_only
2024-09-18 12:47:39,644 - app - INFO - Setting Mode to track
2024-09-18 12:47:39,644 - app - INFO - Stream stopped successfully
2024-09-18 12:47:39,644 - app - INFO - Stopped TrackerManager
2024-09-18 12:47:39,644 - app - INFO - Tracking task cancelled
2024-09-18 12:47:39,645 - app - WARNING - No active stream to stop
2024-09-18 12:47:39,645 - app - INFO - Stopped TrackerManager
2024-09-18 12:47:39,645 - app - INFO - Cleaned up TrackerManager resources
2024-09-18 12:47:39,645 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:47:39,790 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:47:39,790 - app - INFO - Tracker Mode is track
2024-09-18 12:47:39,791 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-18 12:47:39,816 - app - INFO - Initialized TrackerManager with mode: track
2024-09-18 12:47:39,816 - app - INFO - Mode set to track
2024-09-18 12:47:39,816 - app - INFO - Starting tracking in mode: track
2024-09-18 12:48:05,573 - app - INFO - Setting Mode to idle
2024-09-18 12:48:05,573 - app - INFO - Stopped TrackerManager
2024-09-18 12:48:05,573 - app - INFO - Tracking task cancelled
2024-09-18 12:48:05,573 - app - INFO - Stopped TrackerManager
2024-09-18 12:48:05,573 - app - INFO - Cleaned up TrackerManager resources
2024-09-18 12:48:05,573 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:48:05,573 - app - INFO - Mode set to idle
2024-09-18 12:48:46,488 - app - INFO - Cleaning up resources...
2024-09-18 12:48:46,489 - app - INFO - Disconnected from WebSocket server
2024-09-18 12:48:46,489 - app - INFO - AuthManager cleaned up
2024-09-18 12:49:51,321 - app - INFO - Starting the application
2024-09-18 12:49:51,322 - app - INFO - Configuration loaded from configuration/config.json
2024-09-18 12:49:51,322 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-18 12:49:51,322 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 12:49:51,322 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-18 12:49:51,323 - app - INFO - Development environment detected. Skipping SSL configuration.
2024-09-18 12:49:51,323 - app - INFO - Initialized AuthManager for environment: dev
2024-09-18 12:49:51,323 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-18 12:49:51,323 - app - INFO - Setting Mode to idle
2024-09-18 12:49:51,323 - app - INFO - No change in Mode
2024-09-18 12:49:51,398 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:49:51,398 - app - INFO - Successfully obtained new JWT token
2024-09-18 12:49:51,432 - app - INFO - Connected to WebSocket server
2024-09-18 12:49:51,432 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:49:58,485 - app - INFO - Setting Mode to track
2024-09-18 12:49:58,485 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:49:58,540 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:49:58,540 - app - INFO - Tracker Mode is track
2024-09-18 12:49:58,540 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-18 12:49:58,665 - app - INFO - Initialized TrackerManager with mode: track
2024-09-18 12:49:58,665 - app - INFO - Mode set to track
2024-09-18 12:49:58,665 - app - INFO - Starting tracking in mode: track
2024-09-18 12:50:01,670 - app - INFO - received stream command True
2024-09-18 12:50:05,411 - app - INFO - Setting Mode to idle
2024-09-18 12:50:05,411 - app - INFO - Stopped TrackerManager
2024-09-18 12:50:05,411 - app - INFO - Tracking task cancelled
2024-09-18 12:50:05,411 - app - INFO - Stopped TrackerManager
2024-09-18 12:50:05,411 - app - INFO - Cleaned up TrackerManager resources
2024-09-18 12:50:05,412 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:50:05,412 - app - INFO - Mode set to idle
2024-09-18 12:50:09,850 - app - INFO - Setting Mode to track
2024-09-18 12:50:09,850 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:50:09,901 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:50:09,901 - app - INFO - Tracker Mode is track
2024-09-18 12:50:09,901 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-18 12:50:10,026 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-18 12:50:10,026 - app - INFO - Initialized StreamManager with output URL: rtmp://localhost/live/22?key=j_7pMBBcA3lbmznsEGabX6H3gYOHbSYvwh1A5g-s1HA
2024-09-18 12:50:10,026 - app - INFO - Starting stream
2024-09-18 12:50:10,027 - app - INFO - Stream started successfully
2024-09-18 12:50:10,028 - app - INFO - succesfully started stream manager
2024-09-18 12:50:10,028 - app - INFO - Initialized TrackerManager with mode: track
2024-09-18 12:50:10,028 - app - INFO - Mode set to track
2024-09-18 12:50:10,028 - app - INFO - Starting tracking in mode: track
2024-09-18 12:50:30,484 - app - INFO - received stream command False
2024-09-18 12:50:30,485 - app - INFO - Stream stopped successfully
2024-09-18 12:50:30,485 - app - INFO - succesfully stopped stream manager
2024-09-18 12:50:38,392 - app - INFO - Setting Mode to idle
2024-09-18 12:50:38,392 - app - INFO - Stopped TrackerManager
2024-09-18 12:50:38,392 - app - INFO - Tracking task cancelled
2024-09-18 12:50:38,392 - app - INFO - Stopped TrackerManager
2024-09-18 12:50:38,392 - app - INFO - Cleaned up TrackerManager resources
2024-09-18 12:50:38,393 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-18 12:50:38,393 - app - INFO - Mode set to idle
2024-09-18 12:50:46,720 - app - INFO - Cleaning up resources...
2024-09-18 12:50:46,721 - app - INFO - Disconnected from WebSocket server
2024-09-18 12:50:46,721 - app - INFO - AuthManager cleaned up
2024-09-27 07:51:10,206 - app - INFO - Starting the application
2024-09-27 07:51:10,206 - app - INFO - Configuration loaded from configuration/config.json
2024-09-27 07:51:10,208 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-27 07:51:10,208 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-27 07:51:10,208 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-27 07:51:10,208 - app - INFO - Initialized AuthManager for environment: dev
2024-09-27 07:51:10,208 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-27 07:51:10,208 - app - INFO - Setting Mode to idle
2024-09-27 07:51:10,208 - app - INFO - No change in Mode
2024-09-27 07:51:10,209 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:51:10,209 - app - ERROR - 
2024-09-27 07:51:14,413 - app - INFO - Cleaning up resources...
2024-09-27 07:51:14,414 - app - INFO - Disconnected from WebSocket server
2024-09-27 07:51:14,414 - app - INFO - AuthManager cleaned up
2024-09-27 07:51:29,196 - app - INFO - Starting the application
2024-09-27 07:51:29,196 - app - INFO - Configuration loaded from configuration/config.json
2024-09-27 07:51:29,197 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-27 07:51:29,197 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-27 07:51:29,197 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-27 07:51:29,197 - app - INFO - Initialized AuthManager for environment: dev
2024-09-27 07:51:29,197 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-27 07:51:29,197 - app - INFO - Setting Mode to collect_data
2024-09-27 07:51:29,197 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-27 07:51:29,260 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-27 07:51:29,260 - app - INFO - Tracker Mode is collect_data
2024-09-27 07:51:29,260 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-27 07:51:29,388 - app - INFO - Initialized TrackerManager with mode: collect_data
2024-09-27 07:51:29,388 - app - INFO - Mode set to collect_data
2024-09-27 07:51:29,389 - app - INFO - Starting tracking in mode: collect_data
2024-09-27 07:51:29,391 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:51:29,391 - app - ERROR - 
2024-09-27 07:51:34,393 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:51:34,394 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:51:39,395 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:51:39,395 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:51:39,451 - app - INFO - Sending data from tracking loop
2024-09-27 07:51:39,467 - app - ERROR - Error uploading: Constructor parameter should be str
2024-09-27 07:51:40,853 - app - INFO - Cleaning up resources...
2024-09-27 07:51:40,853 - app - INFO - Stopped TrackerManager
2024-09-27 07:51:40,853 - app - INFO - Cleaned up TrackerManager resources
2024-09-27 07:51:40,853 - app - INFO - Disconnected from WebSocket server
2024-09-27 07:51:40,853 - app - INFO - AuthManager cleaned up
2024-09-27 07:51:40,854 - app - INFO - Tracking task cancelled
2024-09-27 07:51:40,854 - app - INFO - Stopped TrackerManager
2024-09-27 07:51:40,854 - app - INFO - Cleaned up TrackerManager resources
2024-09-27 07:52:26,455 - app - INFO - Starting the application
2024-09-27 07:52:26,455 - app - INFO - Configuration loaded from configuration/config.json
2024-09-27 07:52:26,456 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-27 07:52:26,456 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-27 07:52:26,456 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-27 07:52:26,456 - app - INFO - Initialized AuthManager for environment: dev
2024-09-27 07:52:26,456 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-27 07:52:26,456 - app - INFO - Setting Mode to collect_data
2024-09-27 07:52:26,456 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-27 07:52:26,493 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-27 07:52:26,493 - app - INFO - Tracker Mode is collect_data
2024-09-27 07:52:26,493 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-27 07:52:26,618 - app - INFO - Initialized TrackerManager with mode: collect_data
2024-09-27 07:52:26,619 - app - INFO - Mode set to collect_data
2024-09-27 07:52:26,619 - app - INFO - Starting tracking in mode: collect_data
2024-09-27 07:52:26,621 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:52:26,621 - app - ERROR - 
2024-09-27 07:52:31,623 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:52:31,623 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:52:36,625 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:52:36,626 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:52:36,635 - app - INFO - Sending data from tracking loop
2024-09-27 07:52:36,645 - app - ERROR - Error uploading: Constructor parameter should be str
2024-09-27 07:52:37,981 - app - INFO - Cleaning up resources...
2024-09-27 07:52:37,981 - app - INFO - Stopped TrackerManager
2024-09-27 07:52:37,981 - app - INFO - Cleaned up TrackerManager resources
2024-09-27 07:52:37,982 - app - INFO - Disconnected from WebSocket server
2024-09-27 07:52:37,982 - app - INFO - AuthManager cleaned up
2024-09-27 07:52:37,982 - app - INFO - Tracking task cancelled
2024-09-27 07:52:37,982 - app - INFO - Stopped TrackerManager
2024-09-27 07:52:37,982 - app - INFO - Cleaned up TrackerManager resources
2024-09-27 07:54:46,789 - app - INFO - Starting the application
2024-09-27 07:54:46,789 - app - INFO - Configuration loaded from configuration/config.json
2024-09-27 07:54:46,789 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-27 07:54:46,790 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-27 07:54:46,790 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-27 07:54:46,790 - app - INFO - Initialized AuthManager for environment: dev
2024-09-27 07:54:46,790 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-27 07:54:46,790 - app - INFO - Setting Mode to collect_data
2024-09-27 07:54:46,790 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-27 07:54:46,827 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-27 07:54:46,827 - app - INFO - Tracker Mode is collect_data
2024-09-27 07:54:46,827 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-27 07:54:46,954 - app - INFO - Initialized TrackerManager with mode: collect_data
2024-09-27 07:54:46,954 - app - INFO - Mode set to collect_data
2024-09-27 07:54:46,955 - app - INFO - Starting tracking in mode: collect_data
2024-09-27 07:54:46,957 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:54:46,957 - app - ERROR - 
2024-09-27 07:54:51,959 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:54:51,959 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:54:56,958 - app - INFO - Sending data from tracking loop
2024-09-27 07:54:56,967 - app - ERROR - Error uploading: Constructor parameter should be str
2024-09-27 07:54:56,968 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:54:56,968 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:55:01,970 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:55:01,970 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:55:06,958 - app - INFO - Sending data from tracking loop
2024-09-27 07:55:06,965 - app - ERROR - Error uploading: Constructor parameter should be str
2024-09-27 07:55:06,972 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:55:06,972 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 07:55:09,649 - app - INFO - Cleaning up resources...
2024-09-27 07:55:09,649 - app - INFO - Stopped TrackerManager
2024-09-27 07:55:09,649 - app - INFO - Cleaned up TrackerManager resources
2024-09-27 07:55:09,649 - app - INFO - Disconnected from WebSocket server
2024-09-27 07:55:09,649 - app - INFO - AuthManager cleaned up
2024-09-27 07:55:09,649 - app - INFO - Tracking task cancelled
2024-09-27 07:55:09,650 - app - INFO - Stopped TrackerManager
2024-09-27 07:55:09,650 - app - INFO - Cleaned up TrackerManager resources
2024-09-27 08:09:03,509 - app - INFO - Starting the application
2024-09-27 08:09:03,510 - app - INFO - Configuration loaded from configuration/config.json
2024-09-27 08:09:03,510 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-27 08:09:03,510 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-27 08:09:03,510 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-27 08:09:03,510 - app - INFO - Initialized AuthManager for environment: dev
2024-09-27 08:09:03,511 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-27 08:09:03,511 - app - INFO - Setting Mode to idle
2024-09-27 08:09:03,511 - app - INFO - No change in Mode
2024-09-27 08:09:03,512 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:09:03,512 - app - ERROR - 
2024-09-27 08:09:06,757 - app - INFO - Cleaning up resources...
2024-09-27 08:09:06,758 - app - INFO - Disconnected from WebSocket server
2024-09-27 08:09:06,758 - app - INFO - AuthManager cleaned up
2024-09-27 08:09:24,751 - app - INFO - Starting the application
2024-09-27 08:09:24,751 - app - INFO - Configuration loaded from configuration/config.json
2024-09-27 08:09:24,752 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-27 08:09:24,752 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-27 08:09:24,752 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-27 08:09:24,752 - app - INFO - Initialized AuthManager for environment: dev
2024-09-27 08:09:24,752 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-27 08:09:24,752 - app - INFO - Setting Mode to collect_data
2024-09-27 08:09:24,752 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-27 08:09:24,790 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-27 08:09:24,790 - app - INFO - Tracker Mode is collect_data
2024-09-27 08:09:24,790 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-27 08:09:24,915 - app - INFO - Initialized TrackerManager with mode: collect_data
2024-09-27 08:09:24,915 - app - INFO - Mode set to collect_data
2024-09-27 08:09:24,916 - app - INFO - Starting tracking in mode: collect_data
2024-09-27 08:09:24,918 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:09:24,918 - app - ERROR - 
2024-09-27 08:09:29,920 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:09:29,920 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:09:34,921 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:09:34,921 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:09:34,942 - app - INFO - Sending data from tracking loop
2024-09-27 08:09:34,949 - app - ERROR - Error uploading: Constructor parameter should be str
2024-09-27 08:09:39,922 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:09:39,922 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:09:44,924 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:09:44,924 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:09:45,008 - app - INFO - Sending data from tracking loop
2024-09-27 08:09:45,017 - app - ERROR - Error uploading: Constructor parameter should be str
2024-09-27 08:09:46,499 - app - INFO - Cleaning up resources...
2024-09-27 08:09:46,499 - app - INFO - Stopped TrackerManager
2024-09-27 08:09:46,499 - app - INFO - Cleaned up TrackerManager resources
2024-09-27 08:09:46,500 - app - INFO - Disconnected from WebSocket server
2024-09-27 08:09:46,500 - app - INFO - AuthManager cleaned up
2024-09-27 08:09:46,500 - app - INFO - Tracking task cancelled
2024-09-27 08:09:46,500 - app - INFO - Stopped TrackerManager
2024-09-27 08:09:46,500 - app - INFO - Cleaned up TrackerManager resources
2024-09-27 08:10:29,062 - app - INFO - Starting the application
2024-09-27 08:10:29,062 - app - INFO - Configuration loaded from configuration/config.json
2024-09-27 08:10:29,063 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-27 08:10:29,063 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-27 08:10:29,063 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-27 08:10:29,063 - app - INFO - Initialized AuthManager for environment: dev
2024-09-27 08:10:29,063 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-27 08:10:29,063 - app - INFO - Setting Mode to collect_data
2024-09-27 08:10:29,063 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-27 08:10:29,100 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-27 08:10:29,100 - app - INFO - Tracker Mode is collect_data
2024-09-27 08:10:29,100 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-27 08:10:29,226 - app - INFO - Initialized TrackerManager with mode: collect_data
2024-09-27 08:10:29,226 - app - INFO - Mode set to collect_data
2024-09-27 08:10:29,227 - app - INFO - Starting tracking in mode: collect_data
2024-09-27 08:10:29,229 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:10:29,229 - app - ERROR - 
2024-09-27 08:10:34,231 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:10:34,231 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:10:39,233 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:10:39,233 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:10:39,296 - app - INFO - Sending data from tracking loop
2024-09-27 08:10:39,306 - app - ERROR - Error uploading: Constructor parameter should be str
2024-09-27 08:10:42,881 - app - INFO - Cleaning up resources...
2024-09-27 08:10:42,881 - app - INFO - Stopped TrackerManager
2024-09-27 08:10:42,881 - app - INFO - Cleaned up TrackerManager resources
2024-09-27 08:10:42,882 - app - INFO - Disconnected from WebSocket server
2024-09-27 08:10:42,882 - app - INFO - AuthManager cleaned up
2024-09-27 08:10:42,882 - app - INFO - Tracking task cancelled
2024-09-27 08:10:42,882 - app - INFO - Stopped TrackerManager
2024-09-27 08:10:42,882 - app - INFO - Cleaned up TrackerManager resources
2024-09-27 08:11:22,395 - app - INFO - Starting the application
2024-09-27 08:11:22,395 - app - INFO - Configuration loaded from configuration/config.json
2024-09-27 08:11:22,396 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-27 08:11:22,396 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-27 08:11:22,396 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-27 08:11:22,396 - app - INFO - Initialized AuthManager for environment: dev
2024-09-27 08:11:22,396 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-27 08:11:22,396 - app - INFO - Setting Mode to collect_data
2024-09-27 08:11:22,396 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-27 08:11:22,433 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-27 08:11:22,433 - app - INFO - Tracker Mode is collect_data
2024-09-27 08:11:22,433 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-27 08:11:22,559 - app - INFO - Initialized TrackerManager with mode: collect_data
2024-09-27 08:11:22,559 - app - INFO - Mode set to collect_data
2024-09-27 08:11:22,559 - app - INFO - Starting tracking in mode: collect_data
2024-09-27 08:11:22,561 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:11:22,561 - app - ERROR - 
2024-09-27 08:11:27,563 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:11:27,563 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:11:32,565 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:11:32,565 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:11:32,624 - app - INFO - Sending data from tracking loop
2024-09-27 08:11:32,633 - app - ERROR - Error uploading: Constructor parameter should be str
2024-09-27 08:11:34,468 - app - INFO - Cleaning up resources...
2024-09-27 08:11:34,468 - app - INFO - Stopped TrackerManager
2024-09-27 08:11:34,468 - app - INFO - Cleaned up TrackerManager resources
2024-09-27 08:11:34,469 - app - INFO - Disconnected from WebSocket server
2024-09-27 08:11:34,469 - app - INFO - AuthManager cleaned up
2024-09-27 08:11:34,469 - app - INFO - Tracking task cancelled
2024-09-27 08:11:34,469 - app - INFO - Stopped TrackerManager
2024-09-27 08:11:34,469 - app - INFO - Cleaned up TrackerManager resources
2024-09-27 08:12:03,585 - app - INFO - Starting the application
2024-09-27 08:12:03,586 - app - INFO - Configuration loaded from configuration/config.json
2024-09-27 08:12:03,586 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-27 08:12:03,586 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-27 08:12:03,586 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-27 08:12:03,586 - app - INFO - Initialized AuthManager for environment: dev
2024-09-27 08:12:03,587 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-27 08:12:03,587 - app - INFO - Setting Mode to collect_data
2024-09-27 08:12:03,587 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-27 08:12:03,623 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-27 08:12:03,624 - app - INFO - Tracker Mode is collect_data
2024-09-27 08:12:03,624 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-27 08:12:03,749 - app - INFO - Initialized TrackerManager with mode: collect_data
2024-09-27 08:12:03,749 - app - INFO - Mode set to collect_data
2024-09-27 08:12:03,750 - app - INFO - Starting tracking in mode: collect_data
2024-09-27 08:12:03,751 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:12:03,751 - app - ERROR - 
2024-09-27 08:12:08,753 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:12:08,753 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:12:13,755 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:12:13,755 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:12:13,771 - app - INFO - Sending data from tracking loop
2024-09-27 08:12:13,780 - app - ERROR - Error uploading: Constructor parameter should be str
2024-09-27 08:12:16,233 - app - INFO - Cleaning up resources...
2024-09-27 08:12:16,233 - app - INFO - Stopped TrackerManager
2024-09-27 08:12:16,233 - app - INFO - Cleaned up TrackerManager resources
2024-09-27 08:12:16,233 - app - INFO - Disconnected from WebSocket server
2024-09-27 08:12:16,233 - app - INFO - AuthManager cleaned up
2024-09-27 08:12:16,234 - app - INFO - Tracking task cancelled
2024-09-27 08:12:16,366 - app - INFO - Stopped TrackerManager
2024-09-27 08:12:16,366 - app - INFO - Cleaned up TrackerManager resources
2024-09-27 08:14:09,611 - app - INFO - Starting the application
2024-09-27 08:14:09,611 - app - INFO - Configuration loaded from configuration/config.json
2024-09-27 08:14:09,611 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-27 08:14:09,612 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-27 08:14:09,612 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-27 08:14:09,612 - app - INFO - Initialized AuthManager for environment: dev
2024-09-27 08:14:09,612 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-27 08:14:09,612 - app - INFO - Setting Mode to collect_data
2024-09-27 08:14:09,612 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-27 08:14:09,649 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-27 08:14:09,649 - app - INFO - Tracker Mode is collect_data
2024-09-27 08:14:09,649 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-27 08:14:09,775 - app - INFO - Initialized TrackerManager with mode: collect_data
2024-09-27 08:14:09,775 - app - INFO - Mode set to collect_data
2024-09-27 08:14:09,776 - app - INFO - Starting tracking in mode: collect_data
2024-09-27 08:14:09,778 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:14:09,778 - app - ERROR - 
2024-09-27 08:14:14,780 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:14:14,780 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:14:14,804 - app - INFO - Sending data from tracking loop
2024-09-27 08:14:14,812 - app - ERROR - Error uploading: Constructor parameter should be str
2024-09-27 08:14:17,365 - app - INFO - Cleaning up resources...
2024-09-27 08:14:17,365 - app - INFO - Stopped TrackerManager
2024-09-27 08:14:17,366 - app - INFO - Cleaned up TrackerManager resources
2024-09-27 08:14:17,366 - app - INFO - Disconnected from WebSocket server
2024-09-27 08:14:17,366 - app - INFO - AuthManager cleaned up
2024-09-27 08:14:17,366 - app - INFO - Tracking task cancelled
2024-09-27 08:14:17,366 - app - INFO - Stopped TrackerManager
2024-09-27 08:14:17,366 - app - INFO - Cleaned up TrackerManager resources
2024-09-27 08:15:55,723 - app - INFO - Starting the application
2024-09-27 08:15:55,723 - app - INFO - Configuration loaded from configuration/config.json
2024-09-27 08:15:55,724 - app - INFO - Database initialized successfully at tracking_data.db
2024-09-27 08:15:55,724 - app - WARNING - Configuration key 'cloud_function_url' not found. Using default value: None
2024-09-27 08:15:55,724 - app - WARNING - Configuration key 'server_upload_url' not found. Using default value: None
2024-09-27 08:15:55,724 - app - INFO - Initialized AuthManager for environment: dev
2024-09-27 08:15:55,724 - app - INFO - Using urls ws://localhost/sockets/device, http://localhost/api/auth/device_auth
2024-09-27 08:15:55,724 - app - INFO - Setting Mode to collect_data
2024-09-27 08:15:55,724 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-27 08:15:55,762 - app - INFO - Configuration updated and saved to configuration/config.json
2024-09-27 08:15:55,762 - app - INFO - Tracker Mode is collect_data
2024-09-27 08:15:55,762 - app - WARNING - Configuration key 'fps' not found. Using default value: 30
2024-09-27 08:15:55,888 - app - INFO - Initialized TrackerManager with mode: collect_data
2024-09-27 08:15:55,889 - app - INFO - Mode set to collect_data
2024-09-27 08:15:55,889 - app - INFO - Starting tracking in mode: collect_data
2024-09-27 08:15:55,890 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:15:55,891 - app - ERROR - 
2024-09-27 08:16:00,893 - app - ERROR - Error during authentication: Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:16:00,893 - app - ERROR - Cannot connect to host localhost:80 ssl:default [Connect call failed ('127.0.0.1', 80)]
2024-09-27 08:16:00,952 - app - INFO - Sending data from tracking loop
